fabric-datafixer-v0/src/main/java/net/fabricmc/fabric/api/datafixer/FabricDataFixerUtils.java

```diff
@@ -0,0 +1,35 @@
+package net.fabricmc.fabric.api.datafixer;
+
+import java.util.function.BiFunction;
+
+import com.mojang.datafixers.DataFixer;
+import com.mojang.datafixers.schemas.Schema;
+
+import net.fabricmc.fabric.datafixer.mixin.util.SchemasAccessor;
+import net.fabricmc.fabric.impl.datafixer.FabricDataFixerImpl;
+import net.minecraft.util.crash.CrashException;
+
+/**
+ * This class is what registers and creates DataFixers for the game.
+ * <p><b>Please take extreme caution when using these tools as DataFixers directly interface with the world saves and may corrupt world saves.</b></p>
+ * @disclaimer <b>I, i509VCB and the Fabric Team do not take an responsibility for any worlds that are altered, destroyed, corrupted or damaged through the use of the {@link FabricDataFixerUtils} because of improper configuration, mispellings, incorrect syntax, etc.</b>
+ * @author i509
+ *
+ */
+public class FabricDataFixerUtils {
+    
+    static boolean LOCKED = false;
```

Comment:
I would just expose a getter to this in the API.

I know it doesnt stop someone from changing it, but it makes it a bit harder to do by accident.